OPHD/States/MapViewStateTurn.cpp

```diff
@@ -276,9 +277,9 @@ void MapViewState::updateResources()
 			std::clamp(stored.resources[3], 0, 25)
 		};
 
-		stored -= moved;
-		addRefinedResources(moved);
-		stored += moved;
+stored -= moved;
+addRefinedResources(moved);
+stored += moved;
 	}
```

Comment:
What happened to the indentation here?

Reply:
Definitely accidental change. Not sure how/why I missed this when committing. Gonna push a change direct to master over this (since I messed up and accidentally committed a bunch of changes direct to master instead of a new branch, whoops).